src/lib/stats.js

```diff
@@ -74,6 +74,22 @@ exports.stdev = function(data, len, mean) {
     return Math.sqrt(exports.variance(data, len, mean));
 };
 
+/**
+ * median of a finite set of numbers
+ * reference page: https://en.wikipedia.org/wiki/Median#Finite_set_of_numbers
+**/
+exports.median = function(data, len) {
```

Comment:
Isn't this the same as `Lib.interp(data, 0.5)` ?

Reply:
`Lib.interp` is similar except that it doesn't sort the data. Would you prefer that I rewrite `median` to:
```javascript
exports.median = function(data) {
    var b = data.slice().sort();
    retunr exports.interp(b, 0.5);
};

Reply:
Ha. Good eye! You can ignore my comment. Feel free to rewrite `Lib.median` as

```js
exports.median = function(data) {
    var b = data.slice().sort();
    retunr exports.interp(b, 0.5);
};
```

or not.

Reply:
I rewrote it as suggested and tests `Lib.median` against non-sorted values to :lock: down its behavior in 54b3ff93d95eef21660e66c49cde1f50b59b31a2